### PR TITLE
[VM UT] update UT name

### PR DESCRIPTION
**Pull request by Jim8y**
# Description

This pr focus on standardize the UT file name and method name.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Just run the existing unit tests

**Test Configuration**:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tests/Neo.VM.Tests/UT_StackItem.cs [hunk 1]
```diff
@@ -17,10 +17,10 @@
 namespace Neo.Test
 {
     [TestClass]
-    public class UtStackItem
+    public class UT_StackItem
     {
         [TestMethod]
-        public void HashCodeTest()
+        public void TestHashCodeTest()
```
Comment (superboyiii) [inline tests/Neo.VM.Tests/UT_StackItem.cs:23]:
```suggestion
        public void TestHashCode()
```

File: tests/Neo.VM.Tests/UT_StackItem.cs [hunk 2]
```diff
@@ -80,7 +80,7 @@ public void HashCodeTest()
         }
 
         [TestMethod]
-        public void NullTest()
+        public void TestNullTest()
```
Comment (superboyiii) [inline tests/Neo.VM.Tests/UT_StackItem.cs:83]:
```suggestion
        public void TestNull()
```

File: tests/Neo.VM.Tests/UT_StackItem.cs [hunk 3]
```diff
@@ -90,7 +90,7 @@ public void NullTest()
         }
 
         [TestMethod]
-        public void EqualTest()
+        public void TestEqualTest()
```
Comment (superboyiii) [inline tests/Neo.VM.Tests/UT_StackItem.cs:93]:
```suggestion
        public void TestEqual()
```

File: tests/Neo.VM.Tests/UT_StackItem.cs [hunk 4]
```diff
@@ -106,7 +106,7 @@ public void EqualTest()
         }
 
         [TestMethod]
-        public void CastTest()
+        public void TestCastTest()
```
Comment (superboyiii) [inline tests/Neo.VM.Tests/UT_StackItem.cs:109]:
```suggestion
        public void TestCast()
```

File: tests/Neo.VM.Tests/UT_StackItem.cs [hunk 5]
```diff
@@ -187,7 +187,7 @@ public void CastTest()
         }
 
         [TestMethod]
-        public void DeepCopyTest()
+        public void TestDeepCopyTest()
```
Comment (superboyiii) [inline tests/Neo.VM.Tests/UT_StackItem.cs:190]:
```suggestion
        public void TestDeepCopy()
```

File: tests/Neo.VM.Tests/UT_ExecutionContext.cs [hunk 6]
```diff
@@ -17,15 +17,15 @@
 namespace Neo.Test
 {
     [TestClass]
-    public class UtExecutionContext
+    public class UT__ExecutionContext
```
Comment (cschuchardt88) [inline tests/Neo.VM.Tests/UT_ExecutionContext.cs:20]:
```suggestion
    public class UT_ExecutionContext
```

**Review by cschuchardt88 — CHANGES_REQUESTED** [review]

File: tests/Neo.VM.Tests/UT_ScriptBuilder.cs [hunk 7]
```diff
@@ -21,7 +21,7 @@
 namespace Neo.Test
 {
     [TestClass]
-    public class UtScriptBuilder
+    public class UT__ScriptBuilder
```
Comment (shargon) [inline tests/Neo.VM.Tests/UT_ScriptBuilder.cs:24]:
```suggestion
    public class UT_ScriptBuilder
```

File: tests/Neo.VM.Tests/UT_EvaluationStack.cs [hunk 8]
```diff
@@ -19,7 +19,7 @@
 namespace Neo.Test
 {
     [TestClass]
-    public class UtEvaluationStack
+    public class TestEvaluationStack
```
Comment (shargon) [inline tests/Neo.VM.Tests/UT_EvaluationStack.cs:22]:
```suggestion
    public class UT_EvaluationStack
```

**Review by shargon — DISMISSED** [review]
LGTM

**Comment by Jim8y** [comment]
@cschuchardt88

**Review by cschuchardt88 — APPROVED** [review]

**Comment by shargon** [comment]
Wait for merge to https://github.com/neo-project/neo/pull/3082 or it will produce conflicts

**Comment by shargon** [comment]
@Jim8y Conflicts

**Comment by Jim8y** [comment]
@shargon  good to go

**Review by shargon — APPROVED** [review]

**Pull request merged**